tests.py

```diff
@@ -167,6 +167,14 @@ def test_nothing_ends_before_it_starts(self):
                                 "starts." % k
                         )
 
+    def test_reporter_exists(self):
```

Comment:
This just checks that the `T.C.` reporters exist? Do we need this? If so, why not add it for `F.` as well?

Reply:
No- I thought this was removed from the PR.  It is now removed.